internal/integration/postgres_test.go

```diff
@@ -537,6 +536,60 @@ schema "second" {
 	})
 }
 
+func TestPostgres_HCL_ForeignKeyCrossSchema(t *testing.T) {
+	const expected = `table "credit_cards" {
```

Comment:
can't this test be expressed in testscript?

Reply:
sadly no since testscript does not allow to apply on more than one schema (at least for now it does not).

![image](https://user-images.githubusercontent.com/12862103/184114097-e5d830e7-f791-4092-b74b-701936f26f8b.png)